instuction.py

```diff
@@ -0,0 +1,8 @@
+def main():
+    first_user = input("Player 1 name: ")
+    second_user = input("Player 2 name: ")
+    print(f"{first_user} has been assigned a symbol (X) and the {second_user} has been assigned a symbol (O) ")
+    print(f"The game will be played between {first_user} and {second_user}")
+    print("the objective is to make a straight line of three using your assigned symbols by choosing correct numbers and win!")
```

Comment:
You can use a multiple-line syntax to create this string and use only one print statement to display the output. Please refer to [this](https://www.w3schools.com/python/python_strings.asp).

Reply:
ok sure let me work on it